docs/admin/admission-controllers.md

```diff
@@ -190,25 +215,24 @@ Examples of information you might put here are:
 
 In any case, the annotations are provided by the user and are not validated by Kubernetes in any way. In the future, if an annotation is determined to be widely useful, it may be promoted to a named field of ImageReviewSpec.
 
-### ServiceAccount
-
-This plug-in implements automation for [serviceAccounts](/docs/user-guide/service-accounts).
-We strongly recommend using this plug-in if you intend to make use of Kubernetes `ServiceAccount` objects.
+### Initializers (alpha)
 
-### SecurityContextDeny
+This plug-in is related to the [Dynamic Admission Control](/docs/admin/extensible-admission-controllers)
+introduced in v1.7.
```

Comment:
I think you can explain further:

> It determines initializers of a resource based on existing `InitializerConfiguration`s , sets the pending initializers on the resource metadata and blocks the Create request until all pending initializers complete.

this is my understanding, @caesarxuchao to confirm

Reply:
Thanks for the suggestion. Description revised.